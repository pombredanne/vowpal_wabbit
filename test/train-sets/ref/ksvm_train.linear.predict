0.000000
0.533802
0.024339
-0.411951
-0.492665
-0.584747
-0.417858
-0.521428
-0.503527
-0.582914
-0.463158
-0.521759
-0.555031
-0.629764
-0.703308
-0.450767
-0.406695
-0.201802
-0.561377
-0.389729
-0.553136
-0.451236
-0.305807
-0.422035
-0.127303
-0.495621
-0.401073
-0.639340
-0.422899
-0.120011
-0.497806
-0.356542
-0.458424
-0.468078
-0.509064
-0.400358
-0.497955
-0.311618
-0.385408
0.049903
-0.353206
-0.368816
-0.056989
-0.383142
-0.690106
-0.617878
-0.146217
-0.501119
-0.459274
-0.260637
-0.564483
-0.303311
-0.414080
-0.290162
-0.376872
0.309952
-0.309207
-0.556520
-0.453607
-0.266230
-0.412704
-0.078818
-0.033613
-0.617392
-0.188758
-0.112526
-0.312238
-0.273125
-0.563032
-0.387905
-0.051699
-0.205812
-0.341185
-0.660810
0.102876
-0.195549
0.030161
-0.527663
-0.392530
-0.421755
-0.455697
-0.319579
-0.066875
-0.262961
-0.684680
-0.411370
-0.589703
-0.075547
-0.115876
-0.589045
-0.384285
-0.419464
0.440885
-0.310706
0.311892
-0.572667
-0.383448
-0.069443
-0.499425
-0.378976
-0.192021
-0.383127
-0.103490
-0.347129
-0.309629
-0.211496
-0.362474
-0.560248
-0.489263
-0.113483
-0.144653
-0.062826
-0.162182
-0.492239
0.198699
-0.449172
-0.410362
-0.273271
-0.037193
-0.289553
-0.241926
-0.141993
-0.150995
-0.498211
0.491775
0.258556
-0.080804
0.203093
-0.488752
-0.262879
0.026444
-0.025361
-0.029722
-0.312513
-0.295675
-0.759903
-0.194018
-0.041625
0.802205
-0.431106
-0.485976
-0.489143
0.566653
0.741801
-0.053693
0.323181
-0.255489
0.728841
-0.839172
-0.384707
0.206497
-0.297474
0.588321
0.094685
-0.550129
0.077403
0.709433
-0.072283
0.783521
-0.165164
0.601850
-0.408417
-0.228791
0.170817
-0.272026
-0.134575
0.586603
-0.338271
-0.339507
0.539591
0.493736
-0.708862
-0.334013
0.148696
-0.468876
0.008550
-0.706263
0.924668
0.261432
0.736791
0.084992
0.235926
0.153235
-0.374135
-0.250127
-0.259804
0.259987
-0.490107
-0.405003
0.386986
0.674371
0.024151
-0.332929
-0.492281
-0.373955
-0.033944
0.157178
-0.451866
-0.170446
-0.141979
-0.093815
0.012157
0.232594
-0.637752
-0.580518
-0.668803
0.563295
-0.043022
-0.034597
0.025639
-0.064474
-0.788022
-0.285405
0.104404
1.371133
-0.630096
-0.426191
-0.298775
-0.117556
-0.070479
0.177216
-0.487928
-0.492646
-0.366804
0.043868
-0.286433
1.586726
0.199921
-0.391878
-0.700875
-0.693536
-0.744741
-0.760281
-0.024023
0.114491
-0.686013
-0.480271
0.309115
1.041666
0.105215
0.199556
-0.261657
-0.805421
0.696850
-0.073838
0.453353
0.556730
-0.503147
-0.408043
0.827826
