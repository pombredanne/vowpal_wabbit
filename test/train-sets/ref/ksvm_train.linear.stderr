using l2 regularization = 1
predictions = ksvm_train.linear.predict
Lambda = 1
Kernel = linear
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.266901 1.533802            2            2.0  -1.0000   0.5338      103
1.036548 0.806194            4            4.0  -1.0000  -0.4120      134
0.912373 0.788199            8            8.0  -1.0000  -0.5214      145
0.897671 0.882969           16           16.0   1.0000  -0.4508       23
0.922436 0.947200           32           32.0  -1.0000  -0.3565       31
0.921296 0.920156           64           64.0  -1.0000  -0.6174       60
0.917609 0.913923          128          128.0   1.0000   0.2031      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.806317
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 239
Number of kernel evaluations = 140880 Number of cache queries = 228024
Total loss = 201.579147
Done freeing model
Done freeing kernel params
Done with finish 
